com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/CustomRenderTexture/CustomRenderTextureSizeNode.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using UnityEditor.ShaderGraph.GraphDelta;
+
+namespace UnityEditor.ShaderGraph.Defs
+{//TODO: define checkings for preview shader
+    internal class CustomRenderTextureSizeNode : IStandardNode
+    {
+        static string Name = "CustomRenderTextureSize";
+        static int Version = 1;
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Version,
+            Name,
+@"
+TextureHeight = CustomRenderTextureHeightRef;
```

Comment:
Nit - can we put these in the same order as they appear on the node? (Width, Height, Depth)